### PR TITLE
TG-1015 Replace cookiecut project's `black` with `ruff format`

### PR DESCRIPTION
**Pull request by niccolomineo**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -26,8 +23,8 @@ exclude_dirs = [".venv", "tests", "features"]
 
 [tool.ruff]
 extend-exclude = [
-    "__pycache__",
-    ".vscode*",
+    "[{][{]cookiecutter[.]project_dirname[}][}]/__pycache__",
+    "[{][{]cookiecutter[.]project_dirname[}][}]/.vscode*",
```
Comment (niccolomineo) [inline pyproject.toml:27]: These rules currently make `ruff format` error out. Confirmed to be a [bug](https://github.com/astral-sh/ruff/issues/9381) on their side.

**Comment by niccolomineo** [comment]
This PR is currently blocked due to the following pyproject rules making `ruff format` error out:

```
extend-exclude = [
    "__pycache__",
    ".vscode*",
]
```

Confirmed to be a [bug](https://github.com/astral-sh/ruff/issues/9381) on their side.

File: {{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/settings.py [hunk 2]
```diff
@@ -184,6 +184,8 @@ class ProjectDefault(Configuration):
 
     SESSION_COOKIE_SECURE = True
 
+    SESSION_COOKIE_SAMESITE = "Strict"
```
Comment (trottomv) [inline {{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/settings.py:187]: This seems like a customization for a specific project, it doesn't seem necessary to me include it in the bootstrap.

File: {{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/settings.py [hunk 3]
```diff
@@ -378,6 +380,8 @@ def DATABASES(self):  # pragma: no cover
     # Security
     # https://docs.djangoproject.com/en/stable/topics/security/
 
+    CSRF_COOKIE_SAMESITE = "Strict"
```
Comment (trottomv) [inline {{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/settings.py:383]: As mentioned above, also this seems like a customization for a specific project, it doesn't seem necessary to me include it in the bootstrap.

**Review by trottomv — CHANGES_REQUESTED** [review]

**Review by pauloxnet — APPROVED** [review]

**Review by trottomv — APPROVED** [review]

**Pull request merged**